res/mods.js

```diff
@@ -126,7 +142,7 @@ const mods = [
             "modrinth": "https://modrinth.com/mod/fabricated-forge",
         },
         "working": true,
-        "versions": [ "1.4.2", "1.4.1", "1.4", "1.3.2" ]
+        "versions": [ "1.4.7", "1.3.2" ]
```

Comment:
why do you remove minor versions?

Reply:
Minor versions are unnecessary. 
I think the way to show versions needs to be improved, like using version range eg. 1.3.2-1.4.7 or 1.4.X.

Reply:
The current setup is not made to work this way tho, we can do a refactor later in another pr, for now tho just don't remove those stuff.

Reply:
maybe filter with 1.3, 1.4, 1.5... instead of full version is an idea